Doc/reference/lexical_analysis.rst

```diff
@@ -552,7 +552,7 @@ Standard C.  The recognized escape sequences are:
 +-----------------+---------------------------------+-------+
 | Escape Sequence | Meaning                         | Notes |
 +=================+=================================+=======+
-| ``\newline``    | Backslash and newline ignored   |       |
+| ``\`` <LF>      | Backslash and newline ignored   | \(1)  |
```

Comment:
I considered using `:kbd:`, but eventually decided against. The <LF> without markup stands out as separate from the `\` literal, and the <> IMHO help indicate that it's a single entity (that correspond to a line feed) and not a string of two characters.  LF is also used in the same table without special markup (and without <>, but in a context where it's not something you type).

For comparison, this is my original version, where the `<LF>` is easily recognizable as not being part of the literal:
![image](https://user-images.githubusercontent.com/25624924/185800397-fbd5afca-a858-4f7c-a0ab-97dbe004cecf.png)

Your suggestion without space almost looks like `\LF` (at least with the current theme):
![image](https://user-images.githubusercontent.com/25624924/185800264-2cf10354-1735-44d3-9496-868523f7cb08.png)

With the space and `:kbd:` is a bit better:
![image](https://user-images.githubusercontent.com/25624924/185800323-6eddabc3-d8d9-482c-bf82-c4fea503ed69.png)

Reply:
Sorry for the delay—thought I replied to this before.

Ah, I see—on our theme, we render `:kbd:` as something that looks more distinctly like it does on GitHub, i.e.

`\`<kbd>LF</kbd>

This is similar to how control characters are rendered in many/most editors, making it easier for users to infer the intended meaning, uses more appropriate semantic markup, and avoids ambiguous characters that could be interpreted as literals.

However, I inexplicably neglected to check how `:kbd:` renders in the Python docs theme, which of course plays a major role in how this interpreted. As such, it is clear why you made the decision you did—as you say, `:kbd:` makes it much less clear, not more, that `LF` is not part of the literal. I should have checked here, like you did—my mistake, sorry.

So, I agree with your conclusions on the options presented. However, I'm still somewhat unsatisfied with `\` <LF>`—it still has a space that could be interpreted as part of a literal, the `<`/`>` can be potentially ambiguous, and it isn't totally obvious to all users what it is intended to actually be. This perhaps gets at a more meaningful concern—`LF` can be quite cryptic to many users who aren't familiar with the ASCII control characters, and furthermore isn't even really accurate—the newline character could just as well be `CRLF` on Windows platforms, and perhaps even others as well.

How about as a potentially better solution:

`\`<newline>

This:

* Looks much less like a literal, avoiding any confusion and the need for an extra space
* Is much more clear to a user reading it what it means, without having to be well versed in ASCII control codes
* Is more accurate, since what is escaped is any interpreter-recognized newline character, not `LF` specifically

Reply:
```suggestion
| ``\``\ <newline>      | Backslash and newline ignored   | \(1)  |
```

Somehow forgot to make this as an actual suggestion, sorry

Reply:
![image](https://user-images.githubusercontent.com/25624924/186970636-106da0b7-8a03-4f20-a585-3d4244fccc7c.png)
This looks good enough.